Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -1,24 +1,35 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Xml;
 using Newtonsoft.Json;
+using Sdl.Community.ExportAnalysisReports.Interfaces;
 using Sdl.Community.ExportAnalysisReports.Model;
 using Sdl.Community.Toolkit.Core.Services;
 using Sdl.ProjectAutomation.Core;
 using Sdl.ProjectAutomation.FileBased;
 
 namespace Sdl.Community.ExportAnalysisReports.Helpers
 {
-	public static class Help
+	public class Help
```

Comment:
This class should be broken into two separate services, one for Studio and one For Reports and they should have interfaces.

Reply:
I've split the Help class between 2 services which implements interfaces IStudioService and IReportService